app/forms/trn_submission_form.rb

```diff
@@ -52,4 +58,15 @@ def progress_service(progress_key)
   def progress_is_completed?(progress_key)
     progress_service(progress_key).completed?
   end
+
+  def progress_keys
+    keys = []
+    form_validators.each do |validator, options|
+      next if (condition = options[:if]) && !public_send(condition)
```

Comment:
Wondering if the `condition` is necessary...would `next if options[:if] && !public_send(options[:if])` be easier to parse mentally?

Reply:
Personally I thought it would be less cryptic to make it clear you're dealing with a conditional